packages/gasket-plugin-docs/lib/utils/generate-index.js

```diff
@@ -50,11 +49,11 @@ function generateContent(docsConfigSet) {
         : ['Name', 'Description'].concat(additionalHeaders),
       ...docs.map(moduleDoc => {
         const additionalHeaderValues = additionalHeaders.map(h => moduleDoc[h.toLowerCase()]);
-        const { name, description, link, version, targetRoot } = moduleDoc;
-        let itemName = name;
+        const { name, description, link, version, targetRoot, deprecated } = moduleDoc;
+        let itemName = deprecated ? `${name}(deprecated)` : name;
```

Comment:
Minor nit, I think this would look nicer:

```suggestion
        let itemName = deprecated ? `${name} (deprecated)` : name;
```